stdlib/_heapq.pyi

```diff
@@ -9,3 +9,8 @@ def heappop(heap: list[_T], /) -> _T: ...
 def heappush(heap: list[_T], item: _T, /) -> None: ...
 def heappushpop(heap: list[_T], item: _T, /) -> _T: ...
 def heapreplace(heap: list[_T], item: _T, /) -> _T: ...
+def heapify_max(heap: list[Any], /) -> None: ...
+def heappop_max(heap: list[_T], /) -> _T: ...
+def heappush_max(heap: list[_T], item: _T, /) -> None: ...
+def heappushpop_max(heap: list[_T], item: _T, /) -> _T: ...
+def heapreplace_max(heap: list[_T], item: _T, /) -> _T: ...
```

Comment:
You will need `version_info` checks here:

```suggestion
if sys.version_info >= (3, 14):
    def heapify_max(heap: list[Any], /) -> None: ...
    def heappop_max(heap: list[_T], /) -> _T: ...
    def heappush_max(heap: list[_T], item: _T, /) -> None: ...
    def heappushpop_max(heap: list[_T], item: _T, /) -> _T: ...
    def heapreplace_max(heap: list[_T], item: _T, /) -> _T: ...
```

(Same in `heapq.pyi`.)

You will also need to remove the stubtest entries from `stdlib/@tests/stubtest_allowlists/py314.txt`.